scripts/package.json

```diff
@@ -25,7 +25,7 @@
     "github": "^7.0.0",
     "glob": "^7.1.2",
     "gzip-size": "^3.0.0",
-    "jest": "^23.1.0",
+    "jest": "23.0.0",
```

Comment:
Is this necessary? If so, at least peg it to the version we're using currently which is 23.5.0 it looks like from your shrinkwrap change